ksql-parser/src/main/java/io/confluent/ksql/parser/AstBuilder.java

```diff
@@ -572,41 +545,6 @@ public Node visitSingleGroupingSet(SqlBaseParser.SingleGroupingSetContext contex
     );
   }
 
-  @Override
-  public Node visitMultipleGroupingSets(SqlBaseParser.MultipleGroupingSetsContext context) {
```

Comment:
Where did this code come from? Did we just copy the grammar and the parser code wholesale from somewhere? I thought we had just copied the grammar.

Reply:
Most of these are from the unused statements in presto grammar.

Reply:
@apurvam writing you SQL parser from scratch is quite a tedious job. Usually, DB folks prefer to reuse existing one rather write your own. 

https://crate.io/how-we-chose-the-facebook-presto-sql-parser-for-crate-data/